### PR TITLE
# 1.2.0

### PR DESCRIPTION
**Pull request by omerko**
## Type of Change
- [x] Feature Update

## Description of Change
Added USP module support that sends the consent information to Undertone API

### CODE REVIEW

File: undertone/undertone-htb.js [hunk 1]
```diff
@@ -244,12 +244,17 @@ function UndertoneHtb(configs) {
...
+        var uspString = '';
         var gdprConsent = ComplianceService.gdpr.getConsent();
         if (ComplianceService.isPrivacyEnabled() && gdprConsent) {
             gdprValue = gdprConsent.applies ? 1 : 0;
             consentString = gdprConsent.consentString;
         }
-        requestUrl += '&gdpr=' + gdprValue + '&gdprstr=' + consentString;
+
+        if (ComplianceService.isPrivacyEnabled() && ComplianceService.usp.getConsent()) {
+            uspString = ComplianceService.usp.getConsent();
```
Comment (ix-certification) [inline undertone/undertone-htb.js]: Hi,

We seem to have a bad url (http://hb.undertone.com/hb?pid=1&domain=null&gdpr=0&gdprstr=&ccpa=[object%20Object])
getConsent returns an object, the uspString should be ComplianceService.usp.getConsent().uspString

Thanks.

Comment (ix-certification) [reply undertone/undertone-htb.js]: Hi,

The HTTPS change looks good. Please make sure to fix the issue listed above.

Thanks

Comment (ix-certification) [reply undertone/undertone-htb.js]: Hello,

Just an FYI, by default the usp consent is an empty string.

Comment (omerko) [reply undertone/undertone-htb.js]: fixed that, please verify, thanks

**Pull request merged**